distribution/kernel/carbon-home/repository/conf/carbon.xml

```diff
@@ -732,4 +732,7 @@
         </ConfigSync>
     </RemoteLogging>
 
+    <!-- Configure default management console path -->
+    <ManagementConsolePath>carbon</ManagementConsolePath>
```

Comment:
Shouldn't this config say something like DefaultManagementUIPath

Reply:
DefaultServerManagementUIPath?

Reply:
updated